### ISSUE TITLE
Unknown Operation: Merge

### ISSUE THREAD

**Issue by tcco** [riga/tfdeploy#31]
I am trying to deploy a combination of tf.contrib.slim nets including multiple inception_resnet_v1.conv2d and fully_connected that contain an op called Merge. The operation from the model looks like so `InceptionResnetV1/Conv2d_1a_3x3/BatchNorm/cond/Merge`

This operation is currently unsupported and spews this:
`tfdeploy.UnknownOperationException: unknown operation: Merge`

After some digging, the tf.Operation Merge is [tf.summary.merge](https://www.tensorflow.org/api_docs/python/tf/summary/merge). Wanted to get some advice on the viability of adding this operation to tfdeploy? I'd be happy to contribute but wanted to open the issue to begin.
